media/css/firefox/personal.less

```diff
@@ -0,0 +1,333 @@
+         url('/media/fonts/FiraSans-Regular.woff') format('woff'),
+         url('/media/fonts/FiraSans-Regular.ttf') format('truetype');
+    font-weight: normal;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'Fira Sans';
+    src: url('/media/fonts/FiraSans-Bold.eot?#iefix') format('embedded-opentype'),
+         url('/media/fonts/FiraSans-Bold.woff') format('woff'),
+         url('/media/fonts/FiraSans-Bold.ttf') format('truetype');
+    font-weight: bold;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'FiraSans Light';
+    src: url('/media/fonts/FiraSans-Light.eot?#iefix') format('embedded-opentype'),
+         url('/media/fonts/FiraSans-Light.woff') format('woff'),
+         url('/media/fonts/FiraSans-Light.ttf') format('truetype');
+    font-weight: normal;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'FiraSans Light';
+    src: url('/media/fonts/FiraSans-SemiBold.eot?#iefix') format('embedded-opentype'),
+         url('/media/fonts/FiraSans-SemiBold.woff') format('woff'),
+         url('/media/fonts/FiraSans-SemiBold.ttf') format('truetype');
+    font-weight: bold;
+    font-style: normal;
+}
+
+body {
+    font-family: 'Fira Sans'
+}
+
+#outer-wrapper {
+    background: none;
+}
+#wrapper {
+    width: 100%;
+}
+#masthead {
+    background: #0c99d5 url(/media/img/firefox/personal/header.png) 0 0 no-repeat;
+
+    @media only screen and (min-width: @breakTablet) and (max-width: @breakDesktop) {
+        width: 100%;
+    }
+}
+.header-content {
+    margin: 0 auto;
+    padding: 0 0 80px;
+    width: 90%;
+    max-width: 1400px;
+    text-align: center;
+
+    @media only screen and (max-width: @breakMobileLandscape) {
+        padding: 0 0 60px;
+        width: 100%;
+    }
+
+    h1 {
+        color: #bee1f5;
+        margin: 0 0 55px 0;
+        padding-top: 40px;
+        font-family: 'FiraSans Light';
+        .font-size(4.5rem);
+        font-weight: bold;
+        line-height: 1;
+        text-transform: uppercase;
+        letter-spacing: -0.035em;
+        text-shadow: none;
+
+        @media only screen and (max-width: @breakDesktop) {
+            .font-size(3rem);
+        }
+
+        @media only screen and (max-width: @breakTablet) {
+            font-family: 'Fira Sans';
+            .font-size(2.5rem);
+            letter-spacing: -1px;
+        }
+
+        span {
+            display: block;
+            color: #fff;
+            margin-left: 28%;
+
+            @media only screen and (max-width: @breakMobileLandscape) {
+                margin-left: 0;
+            }
+        }
+    }
+    h1:before {
+        display: block;
+        background: transparent url(/media/img/firefox/personal/logo.png) 0 0 no-repeat;
+        background-size: 148px;
+        margin: 0 auto @baseLine;
+        width: 148px;
+        height: 148px;
+        content: '';
+        .at2x('/media/img/firefox/personal/logo.png', 295, 295);
```

Comment:
Looks like you're not supplying a valid unit type here for background-size?

Something like .at2x('/media/img/firefox/personal/logo.png', 148px); should work. Then you shouldn't need the line 103 above.